VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/participant-waiting-room/private-consultation-participants/private-consultation-participants.component.ts

```diff
@@ -113,10 +136,26 @@ export class PrivateConsultationParticipantsComponent extends WRParticipantStatu
         );
     }
 
+    isJohConsultation(): boolean {
+        return this.roomLabel?.toLowerCase().includes('judgejohconsultationroom') ? true : false;
```

Comment:
I see this string is used elsewhere in this class. Can we at least have this as a static readonly property of the class? If this is used elsewhere in the solution we should look at fixing that in the future.

Reply:
judgejohconsultationroom label is given at the api level I think.
It makes sense we bring a static const as a possible refactor for future references